### PR TITLE
843020 UG document for selection zooming support in .NET MAUI 

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description**

Added UG document for selection zooming support

### CODE REVIEW

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 1]
```diff
@@ -209,4 +209,91 @@ chart.ZoomPanBehavior = zooming;
 
 {% endhighlight %}
 
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Selection zooming
+
+Selection zooming feature allows users to interactively choose a particular area of the chart and zoom in. By specifying [EnableSelectionZooming]() property to `true` as shown in the following code snippet, you can double tap and drag to select a range on the chart to be zoomed in.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:216]: Mention Default value of the property

Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md]: default value of the property mentioned

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 2]
```diff
@@ -248,4 +248,132 @@ chart.ZoomPanBehavior = zooming;
...
+
+## Selection zooming
+
+Selection zooming feature allows users to interactively choose a particular area of the chart and zoom in. By specifying [EnableSelectionZooming]() property to `true` as shown in the following code snippet, you can double tap and drag to select a range on the chart to be zoomed in. The default value of this property is false.
+
+{% tabs %}
+
+{% highlight xml %}
+
+<chart:SfCartesianChart>
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:261]: Why code snippet have underline?

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 3]
```diff
@@ -248,4 +248,132 @@ chart.ZoomPanBehavior = zooming;
...
+
+![Selection zooming support in MAUI Chart](Zooming-and-panning_images/maui_selection_zooming.gif)
+
+### Selection rectangle customization
+
+You can customize the selection rectangle using the below properties.
+
+* [SelectionRectStrokeWidth]() - used to change the stroke width of selection rectangle
+
+* [SelectionRectStroke]() - used to change the stroke color of selection rectangle.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:293]: image
To explain the property we used Gets or sets like for all other content so modify similar to this.

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 4]
```diff
@@ -248,4 +248,132 @@ chart.ZoomPanBehavior = zooming;
...
+
+* [SelectionRectStrokeDashArray]() - used to change the stroke dashes of selection rectangle.
+
+* [SelectionRectFill]() - used to change the fill color of selection rectangle.
+
+### Show trackball axis label
+The selection zooming trackball axis label is enabled by setting the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) property to `true`. The default value of the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) is `false`. The trackball axis labels can be customized using following properties:
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:301]: The trackball axis labels can be customized using following properties:
 
 From where user can find this properties?
 
 we not provided support for label alignment. if the support not given remove the property from this list.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Zooming-and-panning.md (hunk 4 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md:301]: mentioned property and removed label alignment property

File: MAUI/Cartesian-Charts/Zooming-and-panning.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md]: underline removed

File: MAUI/Cartesian-Charts/Zooming-and-panning.md (hunk 3 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md]: modified property explanation

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 5]
```diff
@@ -248,4 +248,131 @@ chart.ZoomPanBehavior = zooming;
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Selection zooming
+
+Selection zooming feature allows users to interactively choose a particular area of the chart and zoom in. By specifying [EnableSelectionZooming]() property to `true` as shown in the following code snippet, you can double tap and drag to select a range on the chart to be zoomed in. The default value of this property is false.
+
+{% tabs %}
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:257]: @EzhilarasanElangovan31 Mention how to perform selection zooming on interaction.

Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md:259]: mentioned how to perform selection zooming on interaction

**Review by Saravanan-Madhesh — APPROVED** [review]

**Pull request merged**